Summary/analyze/analyze.py

```diff
@@ -268,65 +268,104 @@ def analyze_library(language, library_name, doc_url, gh_url, domain, repo_path):
 
 def debug_metrics(language, library_name, doc_url, gh_url, domain):
     os.chdir(ROOT_DIR)
```

Comment:
hmm but now this is a different debug_metrics than the one in analyze_library/analyze.py?

Reply:
Yes. I have created a separate directory to add the new revised metrics. Everything under the 'Summary' directory was Henry's code.